src/Notepads/Program.cs

```diff
@@ -35,13 +38,14 @@ static void Main(string[] args)
             {
                 IsPrimaryInstance = true;
                 ApplicationSettingsStore.Write(SettingsKey.ActiveInstanceIdStr, null);
-                instanceHandlerMutex.ReleaseMutex();
```

Comment:
@JasonStein plz make this change in main branch too. When compiled in "Release" mode this line causes some issues, removing this should fix that.

Reply:
What will happen if I remove this?

Reply:
Currently when launching more instances, new instance might not detect the primary instance is present and set itself as primary. But removing this line fixes this. Also, this issue only happens only when app is compiled in release mode.